<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EchoTune</class>
 <widget class="QDialog" name="EchoTune">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Echo tuner</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Main volume (LR)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="echo_mvoll">
        <property name="maximum">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="echo_mvolr">
        <property name="maximum">
         <number>128</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>FIR filter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>0</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>1</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>2</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>3</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>4</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>5</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>6</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>7</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="echo_fir0">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="echo_fir1">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="echo_fir2">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="echo_fir3">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="echo_fir4">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QSpinBox" name="echo_fir5">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QSpinBox" name="echo_fir6">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QSpinBox" name="echo_fir7">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="displayIntegerBase">
         <number>16</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Echo delay</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSpinBox" name="echo_edl">
        <property name="maximum">
         <number>15</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Echo volume (LR)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSpinBox" name="echo_evoll">
        <property name="maximum">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="echo_evolr">
        <property name="maximum">
         <number>128</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="echo_reload">
     <property name="text">
      <string>Reload</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="reset">
     <property name="text">
      <string>Reset to default</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="echo_eon">
     <property name="text">
      <string>Enable echo</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Echo feedback</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSpinBox" name="echo_efb">
        <property name="maximum">
         <number>127</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
